tests/test_attribute_inference_attack.py

```diff
@@ -95,7 +97,27 @@ def test_AIA_on_nursery():
 
     g = GenerateJSONModule("test_attribute_attack.json")
     output = attack_obj.make_report(g)
-    output = output["attack_metrics"]
+    output = output["attack_experiment_logger"]["attack_instance_logger"]["instance_0"]
+
+
+def test_AIA_on_nursery_from_cmd():
+    """tests running AIA on the nursery data
+    with an added continuous feature"""
+    target, _ = common_setup()
+    target.save(path="tests/test_aia_target")
+
+    config = {
+        "n_cpu": 7,
```

Comment:
will 7 CPUs work in general -- need to make sure this script will run anywhere

Reply:
Changed to 2 instead of 7; however, in case we don't pass this to AttributeAttack the default it takes as n_cpu: int = max(1, mp.cpu_count() - 1),